docs/js/main.js

```diff
@@ -36,8 +37,15 @@ const options = {
 // Event listeners
 searchForm.addEventListener('submit', async function (e) {
     e.preventDefault();
-    await getData();
-    showRecipes(recipes);
+    searchMessages.innerText = "Start decreasing your food waste by searching for recipes by ingredient!";
+    let recipeInput = searchBarInput.value.trim();
+    const goodSearch = validateSearch(recipeInput);
+    
+    if (goodSearch) {
+        await getData(recipeInput)
```

Comment:
Since the getData() function grabs the recipeInput itself, you don't need to add it as a parameter here.  But, since you just grabbed it, you could pass it here, but then remove line 78 from the getData() function AND add the parameter in the function declaration in line 76, `async function getData(recipeInput)`